src/pylhe/awkward.py

```diff
@@ -10,12 +10,28 @@ def __dir__():
 
 
 def register_awkward():
-    """Register Awkward behaviors for pylhe."""
+    """This function does nothing and is merely here for backwards compatibility
+    of the pylhe API
 
-    vector.register_awkward()
-    ak.mixin_class(ak.behavior)(Particle)
-    ak.mixin_class(ak.behavior)(Event)
-    ak.mixin_class(ak.behavior)(EventInfo)
+    The registration is done automatically when pylhe is imported since awkward
+    is a required dependency of pylhe.
+
+    .. deprecated:: 0.5.3
+       Simply remove calls since the registration happens automatically now.
+    .. warning:: :func:`~pylhe.register_awkward` will be removed in
+     ``pylhe`` ``v0.7.0``.
+    """
+    import warnings
+
+    warnings.warn(
+        """The `pylhe.register_awkward` function does nothing and has been deprecated
+            as of pylhe v0.5.3 and will be removed in pylhe v0.7.0.
+            The registration now happens automatically when `pylhe` is imported,
+            so calls to `pylhe.register_awkward` can be safely removed.""",
```

Comment:
```suggestion
"pylhe.register_awkward is deprecated as of pylhe v0.6.0 and will be removed in pylhe v0.8.0."
 + " Please remove use of pylhe.register_awkward in favor of automatic registration."
```

No one reads warnings realistically, so if they are going to be read they should be short.